examples/passport/login-with-nextjs/src/app/auth-event-handling/page.tsx

```diff
@@ -0,0 +1,243 @@
+        
+        {isLoggedIn && address && (
+          <Stack gap="space.small">
+            <Body weight="bold">Connected Account:</Body>
+            <code>{address}</code>
+          </Stack>
+        )}
+        
+        {isLoggedIn && chainId && (
+          <Stack gap="space.small">
+            <Body weight="bold">Chain ID:</Body>
+            <code>{chainId}</code>
+          </Stack>
+        )}
+        {accountsState && accountsState.length > 0 && (
+          <Stack gap="space.small">
+            <Body weight="bold">Accounts:</Body>
+            <div>
+              {accountsState.map((account, idx) => (
+                <div key={idx} style={{ marginBottom: "4px" }}>
+                  <code>{account}</code> {idx === 0 && <span style={{ fontSize: "12px", color: "#666" }}>(active)</span>}
+                </div>
+              ))}
+            </div>
+          </Stack>
+        )}
+      </Stack>
+      
+      <Stack direction="row" gap="space.large">
+        <Button 
+          onClick={handleLogin} 
+          disabled={isLoggedIn || loading}
+        >
+          Login {loading && !isLoggedIn ? '...' : ''}
+        </Button>
+        <Button 
+          onClick={handleLogout} 
+          disabled={!isLoggedIn || loading}
+          variant="secondary"
+        >
+          Logout {loading && isLoggedIn ? '...' : ''}
+        </Button>
+      </Stack>
+      
+      {isLoggedIn && (   
+        <Stack direction="row" gap="space.large">
+          <Button 
+            onClick={requestAccounts}
+            disabled={loading}
+            variant="tertiary"
+          >
+            Request Accounts
+          </Button>
+        </Stack>
+      )}
+
+      <Stack gap="space.medium">
+        <Heading size="small">Event Log:</Heading>
+        <div className="event-log" style={{ 
+          maxHeight: '300px', 
+          overflowY: 'auto', 
+          border: '1px solid #ddd',
+          borderRadius: '4px',
+          padding: '12px'
+        }}>
+          {events.length === 0 ? (
+            <Body>No events logged yet</Body>
+          ) : (
+            events.map((event, index) => (
+              <div key={index} style={{ 
+                marginBottom: '12px', 
+                padding: '8px', 
+                backgroundColor: '#f5f5f5',
+                borderRadius: '4px' 
+              }}>
+                <Stack direction="row" gap="space.xsmall" alignItems="center">
+                  <div className="event-tag">{event.event}</div>
+                  <Body size="small" color="secondary">{event.timestamp}</Body>
+                </Stack>
+                <pre style={{ 
+                  marginTop: '4px', 
+                  overflow: 'auto',
+                  fontSize: '12px'
+                }}>
+                  {event.data}
+                </pre>
+              </div>
+            ))
+          )}
+        </div>
+      </Stack>
+    </Stack>
+  );
```

Comment:
Please add the "Return to Examples" link at the bottom like the rest of the examples so there's a way to get back to the feature menu.